test/redirects/redirects.csv

```diff
@@ -37,4 +37,4 @@
 /dora-report-2016,/research/2017-and-earlier/2016-state-of-devops-report.pdf,302
 /dora-report-2015,/research/2017-and-earlier/2015-state-of-devops-report.pdf,302
 /dora-report-2014,/research/2017-and-earlier/2014-state-of-devops-report.pdf,302
-/next24,/survey,302
\ No newline at end of file
+/next24,https://google.qualtrics.com/jfe/form/SV_8uCHA4aRzcGDjg2?source=next24?Q_lang=EN,302
```

Comment:
should we be forcing the languate to `EN`?

shouldn't there be an `&` between `next24` and `Q_lang`, not another `?`

Reply:
Thanks -- fixed